maquetado/scss/details.scss

```diff
@@ -83,3 +82,122 @@
   font-size: $font-normal;
   margin-bottom: 110px;
 }
+
+.back-container {
+  display: flex;
+  height: 100%;
+  justify-content: flex-start;
+  max-height: 416px;
+  max-width: 900px;
+  width: 100%;
+}
+
+.back-arrow-down {
+  display: none;
+}
+
+@media only screen and (max-width: 1024px) {
+  .book-card {
+    box-shadow: 4px 4px 10px $black-russian;
+    display: flex;
+    flex-flow: column;
+    font-family: 'Roboto', sans-serif;
+    height: 100%;
+    justify-content: center;
+    margin: 5px;
+    max-height: 900px;
+    max-width: 416px;
+    padding: 24px;
+    width: 100%;
```

Comment:
You don't need to repeat the classes that were set in desktop.

Reply:
Check it in all the file

Reply:
Done, I remove any duplicate param or class useless (IT mean, each one that don't change nothing). Was that what you requested me, right?